scenes/ui_elements/dialogue/balloon.tscn

```diff
@@ -56,5 +56,7 @@ response_template = NodePath("ResponseExample")
 layout_mode = 2
 text = "Response example"
 
+[node name="TalkSoundPlayer" type="AudioStreamPlayer" parent="."]
```

Comment:
The bus of this AudioStreamPlayer is "master" but there exists 2 other buses: Music and SFX. I guess we want this in the SFX bus.